content/page/get-involved.md

```diff
@@ -0,0 +1,64 @@
+---
+author: kushinauwu
+cover:
+  alt: obf_membership
+  image: /wp-content/uploads/2019/02/obf_membership.jpg
+date: "2019-02-04T19:04:53+00:00"
+guid: https://www.open-bio.org/wp/?page_id=2546
+title: Get Involved
+url: /get-involved/
+aliases:
+  - /membership/
+
+---
+# Join the OBF community
+
+The OBF is a community that invites anyone who is involved in or plans to become involved in promoting open source or open science in a biological field to get involved.
+There are many ways you can become part of our growing community, regardless of your career state, experience with open source/science.
+
+## Join our online spaces
+
+The OBF community has a number of different online spaces that are open to everyone and are used to network and discuss anything related to open bioinformatics. You can:
+
+* [Subscribe to our regular OBF newsletter](https://mailman.open-bio.org/mailman/listinfo/open-bio-l/) (and if you have news to share, you can [also suggest topics](https://github.com/OBF/newsletter/issues).
+* You can [join our community _Slack_](https://join.slack.com/t/open-bio/shared_invite/zt-1pnswao9y-8igcckVxBXhQHCMweHt_NA), where you will find fellow open bioinformatics folks to chat.
+* We also [have an open _LinkedIn_ group](https://www.linkedin.com/groups/9539620/) to get in touch with folks, find interesting job opportunities and more.
```

Comment:
currently, the LinkedIn group is set to allow posting only by moderators. sometimes someone tries to post some ad for something and i delete it. i don't think we want to open up that group to random posts; it will just fill up with ads / spam and then people will leave.